### ISSUE TITLE
Losses should support parameter customization

### ISSUE THREAD

**Issue by petrochenko-pavel-a** [petrochenko-pavel-a/musket_core#1]
Some loss functions such as may depend from hyperparameters:
we need to support this scenarios.
